gammapy/image/models/core.py

```diff
@@ -171,6 +174,134 @@ def evaluate(lon, lat, lon_0, lat_0, sigma):
+        :math:`\text{lat}_0`: `lat` coordinate for the center of the Gaussian.
+    sigma_semi_major : `~astropy.coordinates.Angle`
+        Length of the major semiaxis of the Gaussian, in angular units.
+    e : `float`
+        Eccentricity of the Gaussian (:math:`0< e< 1`).
+    theta : `~astropy.coordinates.Angle`
+        :math:`\theta`:
+        Rotation angle of the major semiaxis.  The rotation angle increases counter-clockwise
+        from the North direction.
+    frame : {"galactic", "icrs"}
+        Coordinate frame of `lon_0` and `lat_0`.
+
+
+    Examples
+    --------
+    .. plot::
+        :include-source:
+
+        import numpy as np
+        import matplotlib.pyplot as plt
+        import astropy.units as u
+        from astropy.coordinates import Angle
+        from gammapy.image.models.core import SkyEllipse
+        from gammapy.maps import Map, WcsGeom
+
+        m_geom = WcsGeom.create(binsz=0.01, width=(5, 5), skydir=(2, 2), coordsys="GAL", proj="AIT")
+        theta = Angle("30 deg")
+        model = SkyGaussianElongated("2 deg", "2 deg", "1 deg", 0.7, theta, frame='galactic')
+
+
+        coords = m_geom.get_coord()
+        lon = coords.lon * u.deg
+        lat = coords.lat * u.deg
+        vals = model(lon, lat)
+        skymap = Map.from_geom(m_geom, data=vals.value)
+
+        _, ax, _ = skymap.smooth("0.05 deg").plot()
+
+        transform = ax.get_transform('galactic')
+        ax.scatter(2, 2, transform=transform, s=20, edgecolor='red', facecolor='red')
+        ax.text(1.5, 1.85, r"$(l_0, b_0)$", transform=transform, ha="center")
+        ax.plot([2, 2 + np.sin(theta)], [2, 2 + np.cos(theta)], color="r", transform=transform)
+        ax.vlines(x=2, color='r', linestyle='--', transform=transform, ymin=-5, ymax=5)
+        ax.text(2.25, 2.45, r"$\theta$", transform=transform);
+        ax.contour(skymap.data, cmap='coolwarm', levels=10, alpha=0.6)
+
+        plt.show()
+    """
+
+    __slots__ = ["frame", "lon_0", "lat_0", "sigma_semi_major", "e", "theta"]
+
+    def __init__(self, lon_0, lat_0, sigma_semi_major, e, theta, frame="galactic"):
+        self.frame = frame
+        self.lon_0 = Parameter(
+            "lon_0", Longitude(lon_0).wrap_at("180d"), min=-180, max=180
+        )
+        self.lat_0 = Parameter("lat_0", Latitude(lat_0), min=-90, max=90)
+        self.sigma_semi_major = Parameter("sigma_semi_major", Angle(sigma_semi_major))
+        self.e = Parameter("e", e, min=0, max=1)
+        self.theta = Parameter("theta", Angle(theta))
+
+        super().__init__(
+            [self.lon_0, self.lat_0, self.sigma_semi_major, self.e, self.theta]
+        )
+
+    @property
+    def evaluation_radius(self):
+        r"""Evaluation radius (`~astropy.coordinates.Angle`).
+
+        Set as :math:`5\sigma`.
+        """
+        return 5 * self.parameters["sigma_semi_major"].quantity
+
+    def evaluate(self, lon, lat, lon_0, lat_0, sigma_semi_major, e, theta):
```

Comment:
Make this a `@staticmethod` and remove `self` as argument, as in all other models.